packages/grafana-ui/src/components/VizLegend/VizLegendTableItem.tsx

```diff
@@ -85,16 +85,16 @@ export const LegendTableItem: React.FunctionComponent<Props> = ({
 
 LegendTableItem.displayName = 'LegendTableItem';
 
-const getStyles = (theme: GrafanaTheme) => {
-  const rowHoverBg = styleMixins.hoverColor(theme.colors.bg1, theme);
+const getStyles = (theme: GrafanaTheme2) => {
+  const rowHoverBg = styleMixins.hoverColor(theme.colors.background.primary, theme);
 
   return {
     row: css`
       label: LegendRow;
-      font-size: ${theme.typography.size.sm};
-      border-bottom: 1px solid ${theme.colors.border1};
+      font-size: ${theme.v1.typography.size.sm};
+      border-bottom: 1px solid ${theme.colors.border.strong};
```

Comment:
border strong seems should only be used for a border hover or other very extreme borders, so this should have border.weak most likely or border medium

Reply:
Oh i see, thanks for the info!